lib/src/participant/local_participant.dart

```diff
@@ -170,7 +170,11 @@ class LocalParticipant extends Participant {
 
   /// Unpublish a track that's already published
   @override
-  Future<void> unpublishTrack(String trackSid, {bool notify = true}) async {
+  Future<void> unpublishTrack(
+    String trackSid, {
+    bool notify = true,
+    bool? stopTrack,
```

Comment:
I wonder if we should offer both defaults *and* a parameter for each unpublish. what if we didn't have a room option; would it change how users would use this?